backend/fms_core/serializers.py

```diff
@@ -273,12 +274,16 @@ class Meta:
         depth = 1
 
 class ProjectSerializer(serializers.ModelSerializer):
+    samples = SampleSerializer(read_only=True, many=True)
     class Meta:
         model = Project
         fields = "__all__"
+        extra_fields = ('samples')
 
 class ProjectExportSerializer(serializers.ModelSerializer):
+    samples = SampleSerializer(read_only=True, many=True)
     class Meta:
         model = Project
-        fields = ("id", "name", "principal_investigator", "requestor_name", "requestor_email", "status", "targeted_end_date",  "comments" )
+        fields = ("id", "name", "principal_investigator", "requestor_name", "requestor_email", "status", "targeted_end_date",  "comments", "samples")
```

Comment:
Same as above (sample pk only) and i would place the samples before the comments.

Reply:
samples are still after comments